src/Tizen.TV.UIControls.Forms/RemoteKeyHandler.cs

```diff
@@ -25,6 +25,8 @@ public class RemoteKeyHandler : BindableObject
     {
         public static readonly BindableProperty CommandProperty = BindableProperty.Create("Command", typeof(ICommand), typeof(RemoteKeyHandler), null);
 
+        RemoteControlKeyTypes? _commandKeyType = null;
```

Comment:
Is it better to make this  as a property?

Reply:
We can make RemoteControlKeyTypes as Flags like below
```c#
    [Flags]
    public enum RemoteControlKeyTypes
    {
        KeyDown = 1,
        KeyUp = 1 << 1
    }
```
And We can use `KeyDown | KeyUp` as Default value
```
RemoteControlKeyTypes _commandKeyType = RemoteControlKeyTypes.KeyDown | RemoteControlKeyTypes.KeyUp;
```